UnitTests/GitUI.Tests/CancellationTokenSequenceTests.cs

```diff
@@ -165,7 +165,10 @@ void ThreadMethod(int i)
 
                     if (countdown.Signal())
                     {
+#pragma warning disable VSTHRD103 // Cancel synchronously blocks. Await CancelAsync instead.
```

Comment:
Why supress instead of changing here?
(Test code, one similar supression, may be ok.)

Reply:
I was making an update in VS Code, and I'm not setup to debug in it. I tried to fix it but it required more changes than I was prepared to spend time for.
It's a test code after all, so no real pressure.